README.md

```diff
@@ -4,40 +4,69 @@ The frontend for the NUSCC Website.
 
 ## 💻 Technologies Used
 
-- **Frontend**: Next.js, React, TypeScript
-- **Styling**: Tailwind CSS, Shadcn UI components
-- **State Management**: React Hooks (useState, useEffect)
-- **Icons**: Lucide React
-- **Date Handling**: date-fns
+- Frontend: Next.js, React, TypeScript
+- Styling: Tailwind CSS, Shadcn UI components
+- State Management: React Hooks (useState, useEffect)
+- Icons: Lucide React
+- Date Handling: date-fns
 
 ## 🚀 Getting Started
 
 ### Prerequisites
 
 - Node.js (v20+)
 - pnpm
+- ngrok account
 
 ### Installation
 
 1. Clone the repository:
 
-```bash
 git clone https://github.com/usdevs/NUSCweb
 cd NUSCweb
-```
 
-1. Install dependencies:
-
-```bash
+2. Install dependencies
 pnpm install
-```
 
-1. Setup your environment variables in `.env` file
+3. Setup your environment variables in .env file
+# Connect to Supabase via connection pooling
+DATABASE_URL="postgresql://postgres.jybpfrxduyftoqptiovt:Zj6gP6K5zAbpQDZQ@aws-1-ap-southeast-1.pooler.supabase.com:6543/postgres?pgbouncer=true"
+
+# Direct connection to the database. Used for migrations
+DIRECT_URL="postgresql://postgres.jybpfrxduyftoqptiovt:Zj6gP6K5zAbpQDZQ@aws-1-ap-southeast-1.pooler.supabase.com:5432/postgres"
+
+BOT_TOKEN= 
+SECRET_KEY=
 
-1. Run the development server:
+NEXT_PUBLIC_TELEGRAM_LOGIN_BOT=
+## leave blank
+NEXT_PUBLIC_POSTHOG_KEY= 
+NEXT_PUBLIC_POSTHOG_HOST=
```

Comment:
Put this in a backtick or something. You could also just say create an `.env` file, with the `.env.example` file as reference, then just explaining each value.